plugins/controlled-documents-resources/src/components/document/DocumentDiffViewer.svelte

```diff
@@ -1,15 +1,15 @@
 <script lang="ts">
   import { getContext, onDestroy } from 'svelte'
-  import { Doc as Ydoc } from 'yjs'
+  import type { Doc as Ydoc } from 'yjs'
```

Comment:
Do not see usage

Reply:
It's there
<img width="679" alt="image" src="https://github.com/user-attachments/assets/a8ec3322-619f-4b6a-9ce4-2a31f357f48e">